3rdparty/openvx/include/openvx_hal.hpp

```diff
@@ -343,11 +361,23 @@ inline int ovx_hal_mul(const T *a, size_t astep, const T *b, size_t bstep, T *c,
 {
     try
     {
+        int rounding_policy = VX_ROUND_POLICY_TO_ZERO;
+        if ((float)scale != 0x1.010102p-8)
```

Comment:
@terfendail , I modified this constant because otherwise it did not compile with GCC. Could you please check whether it works with MSVC?